sdk/resourcemanager/Azure.ResourceManager.Core/docs/Prerequisites.md

```diff
@@ -0,0 +1,55 @@
+Prerequisites
+-------------
+
+You will need the following values to authenticate to Azure
+
+-   **Subscription ID**
```

Comment:
These are *not* prerequisites, except for a single type of authentication.  Having this as the introduction to the client is misleading.

Reply:
For most devops scenaris, what you need is: having logged in previously using some other tool